pulp_ansible/app/viewsets.py

```diff
@@ -245,6 +246,19 @@ class CollectionRemoteViewSet(RemoteViewSet):
     queryset = CollectionRemote.objects.all()
     serializer_class = CollectionRemoteSerializer
 
+    def update(self, request, pk, **kwargs):
+        """Update remote."""
+        if "url" in request.data or "requirements_file" in request.data:
+            repos = list(
+                AnsibleRepository.objects.filter(
+                    remote_id=pk, last_synced_metadata_time__isnull=False
+                ).all()
+            )
+            for repo in repos:
+                repo.last_synced_metadata_time = None
+            AnsibleRepository.objects.bulk_update(repos, ["last_synced_metadata_time"])
+        return super().update(request, pk, **kwargs)
```

Comment:
I think this should all be wrapped in one db transaction.

Reply:
super() triggers a task, not sure if it would be effective to add it